topic-controller/resources/example-topic-cm.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: example-topic-cm
+  labels:
+    app: strimzi
+    kind: topic
```

Comment:
Isn't it using the old way about labeling. Should we remove `app` and changing `kind` to `strimzi.io/kind` ?

Reply:
This file should be deleted, actually: It's not really a resource people should deploy. A follow up PR will be adding an example to the README anyway, so it will end up better documented.